src/plumbing/main.rs

```diff
@@ -226,8 +226,8 @@ pub fn main() -> Result<()> {
                         add_paths: add_path,
                         prefix,
                         files: add_virtual_file
-                            .chunks(2)
-                            .map(|c| (c[0].to_owned(), c[1].clone()))
+                            .chunks_exact(2)
+                            .map(|c| (c[0].clone(), c[1].clone()))
```

Comment:
Let's leave this at chunks as well, it should panic if the count is off (which it will due to indexed access).

Reply:
This one is already checked at the beginning of the function.